modules/ROOT/pages/cloudhub-manage-props.adoc

```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
 To prevent CloudHub properties from overriding properties bundled with the deployable archive, change options in the `property-placeholder` element in the Mule application.
 For more information about nondefault property placeholder options, see the https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/context/support/PropertySourcesPlaceholderConfigurer.html[Spring documentation on Property Placeholder options].
 
+== CloudHub Reserved Properties 
+
+CloudHub reserves a set of properties and prevents apps from overwriting the values of those properties.
+If an application attempts to assign a value to a reserved property, CloudHub doesn't use the assigned value.
+Instead, it returns the value as set internally without throwing an error or exception.
+
+Applications can reference reserved properties, for example, to retrieve the application name or deployment region.
+You can use the `${csorganization.id}` and `${environment.id}` properties to retrieve those values if your app makes Anypoint Platform API calls.
+You can use the `${worker.id}` to identify each worker in a multi-worker application.
+
+--
+[NOTE]
+The `${java.version}` property value doesn't include the point release for the JDK version.
+Additionally, CloudHub runtime release updates might upgrade the JDK.
```

Comment:
Is it obvious how this second sentence is "additional" to the previous? 
It isn't to me, but...that's just me....   ;-)